WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -492,14 +488,14 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
```

Comment:
The layout and code structuring of mail list items and task list items with updated design are different.
It was adding a margin-top, which was  having significance in previous design but now we need more breathing space.